drivers/elum_huawei_smartlogger.json

```diff
@@ -0,0 +1,19 @@
+{
+  "common": {
+    "fncode": 3,
+    "words": 2,
+    "datatype": "float",
+    "order": "lsr"
+  },
+  "fields": {
+    "I_L1": {"register": 2, "unit": "A", "description": "Phase 1 current"},
+    "I_L2": {"register": 4, "unit": "A", "description": "Phase 2 current"},
+    "I_L3": {"register": 6, "unit": "A", "description": "Phase 3 current"},
+    "dc_in_I": {"register": 8, "unit": "A", "description": "DC current"},
+    "dc_in_P": {"register": 28, "unit": "W", "description": "DC Power"},
+    "P": {"register": 106, "unit": "W", "description": "AC total active power"},
```

Comment:
given that we have three phases, should we call this `P_total`, just to avoid ambiguity?